packages/c/creativecommons-vocabulary.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "creativecommons-vocabulary",
+  "filename": "js/vocabulary.js",
+  "description": "A cohesive design system to unite the web facing Creative Commons",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/creativecommons/vocabulary.git"
+  },
+  "keywords": [
+    "ui",
+    "ux",
+    "design",
+    "design system",
+    "components",
+    "sass stylesheets",
+    "library",
+    "monorepo"
+  ],
+  "license": "MIT",
+  "homepage": "https://opensource.creativecommons.org/vocabulary",
+  "autoupdate": {
+    "source": "npm",
+    "target": "@creativecommons/vocabulary",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "**/*.@(js|css|mdx|scss|svg|jpg|png)"
```

```suggestion
          "assets/**/*.@(svg|jpg|png)",
          "css/*.css",
          "scss/*.scss",
          "js/*.js"
```
👋 cdnjs doesn't support `.mdx` files, and I think it'd be nicer to specific which directories we're copying files from as a globstar is involved :)